mslib/msui/qt5/ui_kmloverlay_dockwidget.py

```diff
@@ -2,16 +2,24 @@
 
```

Comment:
I just tested your code. The idea was to have one lw/color field per KML file. Not global buttons. The buttons are not too shabby, but one still cannot see, with one glance, which color belongs to which KML file.

Reply:
@joernu76 can you please guide me through the changes that you want here? or exactly what to add more?

Reply:
I will have another look this afternoon and provide some graphical input to you.

Reply:
The current state looks quite improved.
There is a nice "preview" on the left between the tickbox and the filename showcasing the style. Keep that. The change-colour button changes its color depending on the last selected colour, which does not make sense. Remove that. If a new KML file is added, it should use the defaul color black.

The line width widget is fine.

This can be merged soon.

Reply:
Thankyou! @joernu76 Yeah, I will change the colour to black. (I have thought upon this before also.) And I am working on the changes that you and @Marilyth proposed.